test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/exemptionandlosses/ExemptionAndLossesControllerSpec.scala

```diff
@@ -1809,43 +1809,4 @@ object ExemptionAndLossesControllerSpec extends Matchers {
     )
   }
 
-  def validateExemptionAndLossesCheckYourAnswersPage(
```

Comment:
wasn't this being called in `ExemptionAndLossesControllerSpec` and the spec for the CheckAllAnswersController as well? If they weren't they probably should have been to check that the correct content is being displayed

Reply:
Nope, just copied same function and added one extra param "isAnAgent: Boolean".
You can see that new function just above.
All are using this new function.